docs/source/setup/recipes.rst

```diff
@@ -67,3 +67,222 @@ example, support for rabbitMQ message bus or web support), as well asl supportin
+
+Having configured the system in the last step, we now are able to install and configure the
+VOLTTRON platform. Because this is a user-space process, the ``-K`` flag is not used. This
+command looks like ``ansible-playbook -i <path/to/your/inventory>.yml <path/to/>volttron/deployment/recipes/install-platform.yml``.
+Again, if you review the console output your will see that platform configuration differences
+are reflected.
+
+Having completed this step, many components have been added to the user's home directory on
+the remote systems (or whatever alternate location configured in the inventory file). If you
+ssh to those systems you can inspect those directories and files which have been created.
+
+Step 4: Start the platform
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Starting the platform follows the same pattern as the prior two steps, the command is
+``ansible-playbook -i <path/to/your/inventory>.yml <path/to/>volttron/deployment/recipes/install-platform.yml``.
+This particular playbook simply starts the platform (assuming it is not already running).
+
+If you connect to one of the remote systems, you can source the activation script created
+during the platform installation step. This activates the VOLTTRON environment as well as
+setting appropriate environment variables. If you run ``vctl``, you'll see that the platform
+is running (but currently has no agents installed).
+
+Extra steps
+~~~~~~~~~~~
+
+Having started the platform, you can leverage ansible's ad-hoc commands to interact with them.
+For example, ``ansible -i <path/to/your/inventory>.yml -m shell -a "volttron/env/bin/vctl status"``
+will attempt to run the ``vctl status`` command on each remote (assuming they are all using the
+default VOLTTRON_ROOT location. Similarly, you can override default inventory values from the CLI.
+If you'd like to shutdown the remote platforms you could either set the ``platform_status`` variable
+to ``"stopped"`` in the inventory, or do it on the fly with:
+``ansible-playbook -i <path/to/your/inventory>.yml <path/to/>volttron/deployment/recipes/install-platform.yml -e platform_status=stopped``.
+
+
+.. _recipes-feature-planning:
+
+Feature planning
+----------------
+
+The ansible-based recipes feature set has been long requested, but has some challenges when seeking
+to combine existing VOLTTRON usage patterns and complex state with the normal patterns in ansible
+(especially idempotence). This initial feature set is admittedly not very expansive, but is intended
+to be a starting point and an opportunity to see what usage patterns are most desired. Feedback
+in that regard is greatly appreciated. Our current planning includes the following priorities:
+
+* Support for a multi-platform system with vc connections, specifically creating/authorizing that
+  connection
+
+* Support for working with agents in the installed platforms, eventually to include:
+
+  * installing/uninstalling agents
+
+  * updating configuration of an installed agent
```

Comment:
is this config of the agent or config store for the agent or both?

Reply:
I need to understand how this subsystem works a bit better to know what's possible/expected. Is it the case in general that an agent has a "config" and a "config store" and that these are independent and that they can be interacted with on a running agent?

I think config store is relatively easy to do cleanly because you can check if it already exists by name, and if so you can compare what's there against the requested state and know if you're making a change or not.